site/docs/v1/tech/apis/applications.adoc

```diff
@@ -162,6 +163,125 @@ include::docs/v1/tech/apis/_standard-put-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Search for Applications
+
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Applications and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Applications using the given search and sort criteria
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/application/search``?name=\{name\}&active=\{active\}&tenantId=\{tenantId\}&orderBy={orderBy}&startRow={startRow}``#
+--
+
+==== Request Parameters
+
+[field]#name# [type]#[String]# [optional]#Optional#::
+The string to search for in the Application name. This can contain wildcards using the asterisk or percent characters (* or %). This search parameter is case-insensitive.
```

Comment:
```suggestion
The string to search for in the Application name. This can contain wildcards using the asterisk character (`*`). This parameter is case-insensitive.
```

Reply:
I guess we do support both wildcards for audit logs, but I think it is needlessly complex.

Reply:
I tend to agree with this. Probably simpler to just document one option. 

To note...  if you omit the `*` we will do a "fuzzy" search - for example, if you search for `foo` it will be converted to `%foo%`. But if you search for `foo*` it will be converted to `foo%`, so using a wild card in this example is more specific than omitting it. 

This is just to say, we don't currently have a way to ask a search API to do an exact match like we do in Elasticsearch.

Reply:
@mooreds @robotdan How does this sound?
```The case-insensitive string to search for in the Application name. This can contain wildcards using the asterisk character (*). If no wildcards are present, the search criteria will be interpreted as "*value*". If this field is omitted, no filtering will be applied to the name field.```

Reply:
> If this field is omitted, no filtering will be applied to the name field.

Not sure if we have to say this - if we do, we'd need to add it to all search criteria fields, and I think it is implied.